### PR TITLE
Updating daocreator experimental package to set redirect link from alchemy

### LINKED ISSUE #1948: Wrong URL on xDAI deployment
Right now, when you use the DAO creator on xDAI, after deployment, the links below go to the website: 

https://alchemy-staging-xdai.herokuapp.com/dao/....

It should be going to https://alchemy-xdai.daostack.io/dao/... instead

This is the case for both of these links: http://prntscr.com/tfis10

### PR DESCRIPTION
**Pull request by cbrzn**
I added an example on how you should set the URL via props. Now you can, based on the `targetedNetwork`, define where you want to redirect the user when they create a new DAO.

Note: The props just needs the domain (like in the example I did), the rest is handled by daocreator, you can check the code here: https://github.com/dOrgTech/DAOcreator/blob/issue-509-dynamic_link_on_deployment/packages/ui_v2/src/components/commonV2/dao/Migrator/DeployButton.tsx#L452

### CODE REVIEW

File: src/components/DaoCreator/index.tsx [hunk 1]
```diff
@@ -48,6 +48,7 @@ class DaoCreator extends React.Component<IProps> {
             return await getWeb3Provider();
           }}
           noDAOstackLogo
+          redirectURL="https://alchemy-xdai.daostack.io"
```
Comment (dkent600) [inline src/components/DaoCreator/index.tsx:51]: This does not need to use `targetedNetwork`, can just use `${window.location.protocol}//${window.location.host}` or `process.env.BASE_URL` or some derivation of `RouteComponentProps.location`.  `process.env.BASE_URL` seems easiest.  Whatever the choice, I think the change should be done in this PR.

Comment (cbrzn) [reply src/components/DaoCreator/index.tsx]: You call sir! @dkent600

Comment (dkent600) [reply src/components/DaoCreator/index.tsx]: @cbrzn Cool.  Did you test this?  If yes and it works, I'll approve the PR

Comment (cbrzn) [reply src/components/DaoCreator/index.tsx]: Yup yup @dkent600 - Would if awesome if you can test locally too :-D. I will mark this conv as resolved :+1:

**Comment by orenyodfat** [comment]
I tested that . looks ok.

**Comment by dkent600** [comment]
Hi @cbrzn Looks like there are merge conflicts

**Comment by cbrzn** [comment]
hola! @dkent600 fixed :+1:

**Pull request merged**